src/vs/workbench/contrib/terminal/browser/terminalEditorService.ts

```diff
@@ -19,6 +19,8 @@ export class TerminalEditorService extends Disposable implements ITerminalEditor
 	private _editorInputs: Map</*instanceId*/number, TerminalEditorInput> = new Map();
 	private _instanceDisposables: Map</*instanceId*/number, IDisposable[]> = new Map();
 
+	private _createInstance!: () => ITerminalInstance;
```

Comment:
Let's make this ? and handle the missing `_createInstance` call by throwing in `TerminalEditorInput.copy`.